wnprc_billing/resources/queries/wnprc_billing/groupCategoryAssocWithCategoryName.query.xml

```diff
@@ -0,0 +1,17 @@
+<query xmlns="http://labkey.org/data/xml/query">
+    <metadata>
+        <tables xmlns="http://labkey.org/data/xml">
+            <table tableName="groupCategoryAssocWithCategoryName" tableDbType="NOT_IN_DB">
+                <auditLogging>DETAILED</auditLogging>
+                <columns>
+                    <column columnName="chargeGroupName">
+                        <columnTitle>chargeGroupName</columnTitle>
```

Comment:
Do we really want this to be "chargeGroupName"? And not "Charge Group Name" or something a little more user-oriented?

Reply:
User-Friendly title now set (with required refactor in the trigger script).